src/plugins/intel_gpu/src/graph/include/primitive_inst.h

```diff
@@ -403,6 +403,7 @@ class primitive_inst {
     bool _can_share_buffer = true;
     bool _is_constant = false;
     bool _needs_completion_event = false;
+    bool _no_execution_prev = false;
```

Comment:
If this is used only in prepare_primitive & realloc_if_needed, , could you please remove this from member of primitive_inst, and declare it as a local variable in prepare_primitive, and pass it as function argument of realloc_if_neede?

Reply:
Also, about the name of the variable, how about prev_execution_skipped?

Reply:
Fixed it. please review it again.